packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m

```diff
@@ -277,7 +277,7 @@ + (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
 - (instancetype)init {
   self = [super init];
   if (self) {
-    if (![FIRApp defaultApp]) {
+    if (![FIRApp appNamed:@"__FIRAPP_DEFAULT"]) {
       [FIRApp configure];
```

Comment:
This is definitely an improvement but I think the new log message is still a little confusing. Could we log out a warning about how the user needs to set up their app and to check the readme in the various else cases here?

Reply:
There's no action for the user to take here. It's going to happen for every Flutterfire user, regardless of whether they are set up or not. I wish that we could suppress the log message that the native SDK is sending, but we don't have a way to do that -- see https://github.com/firebase/firebase-ios-sdk/pull/1003

Reply:
I do agree with changing the error message to add something that "This is not a problem if you are using flutter."
I just took 20mins to understand this following threads - and at least liked the response from @collinjackson that "there's no action for the user... It is going to happen for every Flutterfire user, regardless of..."

Reply:
I added an additional log message explaining what is happening when Flutterfire automatically configures the default Firebase app. @mklim let me know if you like this version or if you prefer the previous.

Reply:
Ah understood, thanks. Can you paraphrase your comment and add it into the changelogs or commit message? That explains this a lot more clearly. This new log LGTM.